client/modules/IDE/components/Header/Nav.jsx

```diff
@@ -139,12 +146,19 @@ const ProjectMenu = () => {
   return (
     <ul className="nav__items-left">
       <li className="nav__item-logo">
-        <LogoIcon
-          role="img"
-          aria-label={t('Common.p5logoARIA')}
-          focusable="false"
-          className="svg__logo"
-        />
+        <a
+          href="https://p5js.org/"
+          target="_blank"
+          rel="noreferrer"
+          className="nav__item-logo-link"
+        >
+          <LogoIcon
+            role="img"
+            aria-label={t('Common.p5logoARIA')}
+            focusable="false"
+            className="svg__logo"
+          />
+        </a>
       </li>
```

Comment:
It's kind of annoying that we have this in two places.  It could be extracted into a component.
```
const NavLogo = () => (
  <li className="nav__item-logo">
    <a
      href="https://p5js.org/"
      target="_blank"
      rel="noreferrer"
      className="nav__item-logo-link"
    >
      <LogoIcon
        role="img"
        aria-label={t('Common.p5logoARIA')}
        focusable="false"
        className="svg__logo"
      />
    </a>
  </li>
);
```

Reply:
We can create the component as well but it's just for the logo. Should we really create a component for this?

If yes, then the component file will be in the `client/components/Nav` directory, right?